### ISSUE TITLE
Reopening Dbeaver on startup rises 'unable to create part' error for various tabs

### ISSUE THREAD

**Issue by majkinetor** [serge-rider/dbeaver#663]
Dbeaver version: 3.7.2
OS: any

```
Factory org.jkiss.dbeaver.ui.editors.DatabaseEditorInputFactory returned null 
from createElement for editor id=org.jkiss.dbeaver.ui.editors.entity.EntityEditor name=null
```

It happens for schema view and table view for example but doesn't for SQL Editor window. It happens both in MS SQL and Oracle databases.

Details on schema view for the schema named 'MOF':

```
org.jkiss.dbeaver.DBException: Node 'oracle_thin-15607ba73ee-6a4f3169861b89ec/org.jkiss.dbeaver.ext.oracle.model.OracleSchema/MOF' not found
	at org.jkiss.dbeaver.ui.editors.DatabaseEditorInputFactory$1$1.onTaskFinished(DatabaseEditorInputFactory.java:126)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:126)
	at org.jkiss.dbeaver.ui.editors.DatabaseEditorInputFactory$1.run(DatabaseEditorInputFactory.java:115)
	at org.jkiss.dbeaver.core.DBeaverUI$4.run(DBeaverUI.java:253)
	at org.jkiss.dbeaver.runtime.RunnableContextDelegate$1.run(RunnableContextDelegate.java:45)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)

```

**Comment by serge-rider** [comment]
In most cases it is not a bug - DBeaver just can't restore database object editor because it can't connect or database structure was changed.
SQL Editor do not have any required meta object - that's because it can live even without connection.
However current behavior doesn't look very good (such hard to understand errors are really confusing).

Do you have any workaround on mind?

**Comment by majkinetor** [comment]
> it can't connect or database structure was changed.

This happens on restart, when nothing is changed.

What is the purpose then of the feature '_Database->Reopen editors on startup'_ when it is unusable ?

So that is werid:

- SQL editors **connect to the source as you click the tab**. I don't know what you mean by not haveing any meta object, they clearly remember the source and reconnect to it.
- Other editors do not, but show this error. While you claim they have meta object, it doesn't seem to be used for this purpose.

Solutions:
- Remove option 'Database->Reopen editors on startup' and behave like its not set always.
- Make table/schema viewer remember connection and behave like SQL editors.

**Comment by serge-rider** [comment]
Hmm, sounds like a bug - but I can't reproduce it. If connection is fine then editor should be reopened with no errors.
Table/schema viewers already remembers their connection. I'm using this feature all the time so I definitely don't want to disable it.

Have any ideas how can I reproduce this problem? 
Can you send full error log (from DBeaver startup till editor opening error)?

**Comment by majkinetor** [comment]
Here is the gifshot:

![2016-08-15_11-11-21](https://cloud.githubusercontent.com/assets/85767/17660311/e180de6e-62d8-11e6-8098-cf28e172531b.gif)


Here is the exported log:

```

!ENTRY org.eclipse.osgi 4 0 2016-08-15 11:13:35.419
!MESSAGE Application error
!STACK 1
org.eclipse.swt.SWTException: Failed to execute runnable (java.lang.NullPointerException)
	at org.eclipse.swt.SWT.error(SWT.java:4491)
	at org.eclipse.swt.SWT.error(SWT.java:4406)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:138)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4155)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3772)
	at org.eclipse.swt.widgets.Display.release(Display.java:3825)
	at org.eclipse.swt.graphics.Device.dispose(Device.java:295)
	at org.jkiss.dbeaver.core.application.DBeaverApplication.start(DBeaverApplication.java:206)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:380)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:235)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:669)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:608)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1515)
Caused by: java.lang.NullPointerException
	at org.jkiss.dbeaver.ui.UIUtils.updateMainWindowTitle(UIUtils.java:945)
	at org.jkiss.dbeaver.ui.perspective.DataSourceManagementToolbar.setActivePart(DataSourceManagementToolbar.java:292)
	at org.jkiss.dbeaver.ui.perspective.DataSourceManagementToolbar$9.run(DataSourceManagementToolbar.java:840)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:35)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:135)
	... 17 more

!ENTRY org.eclipse.e4.ui.workbench 4 0 2016-08-15 11:13:35.434
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.eclipse.e4.core.di.InjectionException: org.eclipse.swt.SWTException: Invalid thread access
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:68)
	at org.eclipse.e4.core.internal.di.InjectorImpl.processAnnotated(InjectorImpl.java:898)
	at org.eclipse.e4.core.internal.di.InjectorImpl.disposed(InjectorImpl.java:398)
	at org.eclipse.e4.core.internal.di.Requestor.disposed(Requestor.java:148)
	at org.eclipse.e4.core.internal.contexts.ContextObjectSupplier$ContextInjectionListener.update(ContextObjectSupplier.java:78)
	at org.eclipse.e4.core.internal.contexts.TrackableComputationExt.update(TrackableComputationExt.java:111)
	at org.eclipse.e4.core.internal.contexts.TrackableComputationExt.handleInvalid(TrackableComputationExt.java:74)
	at org.eclipse.e4.core.internal.contexts.EclipseContext.dispose(EclipseContext.java:174)
	at org.eclipse.e4.core.internal.contexts.EclipseContext.dispose(EclipseContext.java:159)
	at org.eclipse.e4.core.internal.contexts.osgi.EclipseContextOSGi.dispose(EclipseContextOSGi.java:106)
	at org.eclipse.e4.core.internal.contexts.osgi.EclipseContextOSGi.bundleChanged(EclipseContextOSGi.java:139)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:902)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:148)
	at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEventPrivileged(EquinoxEventPublisher.java:165)
	at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:75)
	at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:67)
	at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor.publishModuleEvent(EquinoxContainerAdaptor.java:102)
	at org.eclipse.osgi.container.Module.publishEvent(Module.java:466)
	at org.eclipse.osgi.container.Module.doStop(Module.java:624)
	at org.eclipse.osgi.container.Module.stop(Module.java:488)
	at org.eclipse.osgi.container.SystemModule.stop(SystemModule.java:186)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$SystemBundle$EquinoxSystemModule$1.run(EquinoxBundle.java:159)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.eclipse.swt.SWTException: Invalid thread access
	at org.eclipse.swt.SWT.error(SWT.java:4491)
	at org.eclipse.swt.SWT.error(SWT.java:4406)
	at org.eclipse.swt.SWT.error(SWT.java:4377)
	at org.eclipse.swt.widgets.Display.error(Display.java:1258)
	at org.eclipse.swt.widgets.Display.checkDevice(Display.java:764)
	at org.eclipse.swt.widgets.Display.removeFilter(Display.java:4073)
	at org.eclipse.e4.ui.workbench.renderers.swt.MenuManagerRenderer.contextDisposed(MenuManagerRenderer.java:338)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:56)
	... 23 more
!SESSION 2016-08-15 11:14:26.224 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_77
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.ui 4 4 2016-08-15 11:14:29.171
!MESSAGE Unable to create part
!SUBENTRY 1 org.eclipse.ui 4 0 2016-08-15 11:14:29.171
!MESSAGE Factory org.jkiss.dbeaver.ui.editors.DatabaseEditorInputFactory returned null from createElement for editor id=org.jkiss.dbeaver.ui.editors.entity.EntityEditor name=null
```

**Comment by majkinetor** [comment]
As for reproduction, I don't know, but it happened on all of computers I tried it, must be around 10. If you have any instructions for me I will be happy to help.

**Comment by serge-rider** [comment]
I see that schema MOF is missing in the first connection (filtered out?). I think that's why DBeaver can't find it in metadata and can't reopen schema viewer.
Could you explain your config (filters, available schema list, etc)? I'll try to make similar config in my test environment.

**Comment by majkinetor** [comment]
Both have schema filter that contains word 'MOF'. Removing the schema filter doesn't make a difference.
One MOF referes to the database, the other one to the user:

1. The _Oracle - TestBiz - mof/sys_ connection that you refer does not connect to MOF schema but MOF database (SID) using the user sys.
2. The _Oracle - TestBiz - pdbtest1/mof_ for which I actually open the table editor connects to database pdbtest1 (Service Name) using the user mof.  

Here is the data source for the 2 (IP and pass replaced):

```
<data-source id="oracle_thin-1564b32e015-4a47ffea309931c3" provider="oracle" driver="oracle_thin" name="Oracle - TestBiz - pdbtest1/mof" create-date="1470140440597" login-date="1471259499482" save-password="true" show-system-objects="true" read-only="false">
		<connection host="10.x.x.x" port="1521" server="" database="pdbtest1" url="jdbc:oracle:thin:@//10.x.x.x:1521/pdbtest1" user="mof" password="******" home="C:\Oracle\Middleware\Oracle_Home" type="test">
			<property name="@dbeaver-sid-service@" value="SERVICE"/>
			<property name="@dbeaver-use-rule-hint@" value="false"/>
			<property name="@dbeaver-connection-type@" value="BASIC"/>
			<property name="@dbeaver-check-schema-content@" value="true"/>
			<property name="loginTimeout" value="20"/>
			<property name="@dbeaver-always-show-dba@" value="false"/>
			<network-handler type="TUNNEL" id="ssh_tunnel" enabled="false" user="" save-password="false">
				<property name="port" value="22"/>
				<property name="keyPath" value=""/>
				<property name="host" value=""/>
				<property name="authType" value="PASSWORD"/>
			</network-handler>
			<network-handler type="PROXY" id="socks_proxy" enabled="false" user="" save-password="false">
				<property name="socks-host" value=""/>
				<property name="socks-port" value="1080"/>
			</network-handler>
			<bootstrap txnIsolation="2"/>
		</connection>
		<filters>
			<filter type="org.jkiss.dbeaver.model.struct.rdb.DBSSchema">
				<include name="mof"/>
			</filter>
			<filter type="org.jkiss.dbeaver.ext.oracle.model.OracleTable" id="MOF"/>
			<filter type="org.jkiss.dbeaver.ext.oracle.model.OracleSchema">
				<include name="mof"/>
			</filter>
		</filters>
	</data-source>
```

**Comment by serge-rider** [comment]
Still can't reproduce.
Could you try this: 
- open SQL editor for the same connection as for schema viewer. 
- open schema viewer
- make SQL editor tab active
- shutdown DBeaver
- start DBeaver. It will connect to the database
- now click on the schema viewer.

Does the same error happen again?

**Comment by majkinetor** [comment]
Not the same error:
```
org.jkiss.dbeaver.DBException: Node 'oracle_thin-1564b32e015-4a47ffea309931c3/org.jkiss.dbeaver.ext.oracle.model.OracleSchema/MOF' not found
	at org.jkiss.dbeaver.ui.editors.DatabaseEditorInputFactory$1$1.onTaskFinished(DatabaseEditorInputFactory.java:126)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:126)
	at org.jkiss.dbeaver.ui.editors.DatabaseEditorInputFactory$1.run(DatabaseEditorInputFactory.java:115)
	at org.jkiss.dbeaver.core.DBeaverUI$4.run(DBeaverUI.java:253)
	at org.jkiss.dbeaver.runtime.RunnableContextDelegate$1.run(RunnableContextDelegate.java:45)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)
```

![2016-08-15_14-57-25](https://cloud.githubusercontent.com/assets/85767/17665044/a2a0989a-62f8-11e6-8f4f-07c056c93d9f.gif)

**Comment by serge-rider** [comment]
Please send full debug log (https://github.com/serge-rider/dbeaver/wiki/Log-files) on serge@jkiss.org

**Comment by majkinetor** [comment]
Will show it here since there is nothing particularly secret. The log is from the next start once I do what you asked at https://github.com/serge-rider/dbeaver/issues/663#issuecomment-239793891

```
2016-08-15 15:53:35.504 - > Start org.eclipse.e4.ui.workbench [1.3.1.v20160203-0951]
2016-08-15 15:53:35.505 - > Start com.ibm.icu.base [54.1.1.v201501272100]
2016-08-15 15:53:35.505 - > Start org.eclipse.emf.common [2.11.1.v20160208-0816]
2016-08-15 15:53:35.505 - > Start org.eclipse.e4.ui.model.workbench [1.1.100.v20150407-1430]
2016-08-15 15:53:35.506 - > Start org.eclipse.e4.core.di [1.5.0.v20150421-2214]
2016-08-15 15:53:35.533 - > Start org.eclipse.core.filesystem [1.5.0.v20150725-1910]
2016-08-15 15:53:35.576 - > Start org.eclipse.core.resources [3.10.1.v20150725-1910]
2016-08-15 15:53:35.581 - > Start org.eclipse.ui.editors [3.9.0.v20150213-1939]
2016-08-15 15:53:35.620 - > Start org.eclipse.ui.workbench.texteditor [3.9.100.v20141023-1946]
2016-08-15 15:53:35.645 - > Start org.jkiss.dbeaver.core [3.7.2]
2016-08-15 15:53:35.647 - DBeaver Enterprise 3.7.2 is starting
2016-08-15 15:53:35.647 - Install path: 'file:/C:/Program Files/DBeaver/'
2016-08-15 15:53:35.647 - Instance path: 'file:/C:/Users/mmilic/.dbeaver/'
2016-08-15 15:53:35.648 - Memory available 123Mb/1812Mb
2016-08-15 15:53:35.651 - Starting RMI server at 47172
2016-08-15 15:53:35.701 - Initialize display
2016-08-15 15:53:35.719 - > Start org.eclipse.core.databinding.observable [1.5.0.v20150422-0725]
2016-08-15 15:53:35.723 - > Start org.eclipse.e4.ui.workbench.swt [0.13.0.v20150504-0621]
2016-08-15 15:53:35.726 - > Start org.eclipse.e4.core.services [2.0.0.v20150403-1912]
2016-08-15 15:53:35.727 - > Start org.eclipse.e4.core.contexts [1.4.0.v20150828-0818]
2016-08-15 15:53:35.733 - > Start org.eclipse.e4.ui.services [1.2.0.v20150422-0725]
2016-08-15 15:53:35.761 - > Start org.eclipse.equinox.event [1.3.100.v20140115-1647]
2016-08-15 15:53:35.765 - > Start org.eclipse.e4.ui.di [1.1.0.v20150422-0725]
2016-08-15 15:53:35.784 - > Start org.eclipse.emf.ecore [2.11.2.v20160208-0816]
2016-08-15 15:53:35.833 - > Start org.eclipse.emf.ecore.xmi [2.11.1.v20160208-0816]
2016-08-15 15:53:36.063 - > Start org.eclipse.e4.ui.workbench.addons.swt [1.2.0.v20150512-1414]
2016-08-15 15:53:36.070 - > Start org.eclipse.e4.core.commands [0.11.0.v20150422-0725]
2016-08-15 15:53:36.074 - > Start org.eclipse.core.expressions [3.5.0.v20150421-2214]
2016-08-15 15:53:36.091 - > Start org.eclipse.e4.ui.workbench.renderers.swt [0.13.0.v20150901-1347]
2016-08-15 15:53:36.169 - > Start org.eclipse.e4.ui.bindings [0.11.0.v20150422-0725]
2016-08-15 15:53:36.206 - > Start org.eclipse.e4.core.di.extensions [0.13.0.v20150421-2214]
2016-08-15 15:53:36.227 - > Start org.eclipse.ui.ide.application [1.1.0.v20150422-0725]
2016-08-15 15:53:36.463 - > Start org.eclipse.core.filebuffers [3.5.500.v20140723-1040]
2016-08-15 15:53:36.514 - > Start org.eclipse.e4.ui.css.swt [0.12.0.v20150511-1003]
2016-08-15 15:53:36.535 - > Start org.eclipse.ui.ide [3.11.0.v20150825-2158]
2016-08-15 15:53:36.564 - > Start org.eclipse.e4.ui.css.swt.theme [0.10.0.v20150422-0725]
2016-08-15 15:53:36.655 - > Start org.eclipse.ui.forms [3.6.200.v20150506-2029]
2016-08-15 15:53:36.861 - > Start org.eclipse.help [3.6.0.v20130326-1254]
2016-08-15 15:53:36.868 - Initializing DBeaver Enterprise 3.7.2
2016-08-15 15:53:36.869 - Host plugin: com.jkiss.dbeaver.ent.application 3.7.2
2016-08-15 15:53:36.869 - Initialize Core...
2016-08-15 15:53:36.918 - > Start org.eclipse.team.core [3.7.100.v20150203-1452]
2016-08-15 15:53:37.011 - > Start com.jkiss.dbeaver.ent.qmdb [1.0.27]
2016-08-15 15:53:37.013 - Opening QMDB connection...
2016-08-15 15:53:37.153 - Check QMDB deployment
2016-08-15 15:53:37.163 - QMDB started
2016-08-15 15:53:37.164 - Core initialized (294ms)
2016-08-15 15:53:37.164 - Configure workbench window
2016-08-15 15:53:37.610 - Initialize workbench window
2016-08-15 15:53:37.628 - Finish initialization
2016-08-15 15:53:37.698 - > Start org.eclipse.ui.views [3.8.0.v20150422-0725]
2016-08-15 15:53:38.090 - > Start org.eclipse.compare.core [3.5.500.v20150505-1058]
2016-08-15 15:53:38.596 - > Start org.eclipse.equinox.p2.core [2.4.0.v20150527-1706]
2016-08-15 15:53:38.597 - > Start org.eclipse.equinox.p2.operations [2.4.100.v20150423-1455]
2016-08-15 15:53:38.600 - > Start org.eclipse.equinox.p2.engine [2.4.0.v20150511-1532]
2016-08-15 15:53:38.712 - > Start org.eclipse.equinox.p2.metadata [2.3.0.v20150511-1532]
2016-08-15 15:53:38.714 - > Start org.eclipse.equinox.p2.repository [2.3.100.v20150428-1613]
2016-08-15 15:53:38.716 - > Start org.eclipse.equinox.p2.metadata.repository [1.2.200.v20150428-1613]
2016-08-15 15:53:38.722 - > Start org.eclipse.equinox.p2.ui.sdk.scheduler [1.2.100.v20150423-1455]
2016-08-15 15:53:38.723 - > Start org.eclipse.equinox.p2.updatechecker [1.1.200.v20131119-0908]
2016-08-15 15:53:38.756 - Connect with 'Oracle - TestBiz - pdbtest1/mof' (oracle_thin-1564b32e015-4a47ffea309931c3)
2016-08-15 15:53:38.757 - > Start org.jkiss.dbeaver.ext.oracle [2.1.23]
2016-08-15 15:53:38.776 - Unrecognized Oracle client version at C:\Oracle\Middleware\Oracle_Home
2016-08-15 15:53:39.584 - Connected (oracle_thin-1564b32e015-4a47ffea309931c3)
2016-08-15 15:53:40.621 - > Start org.eclipse.gef [3.10.1.201508170204]
2016-08-15 15:53:40.622 - > Start org.jkiss.dbeaver.ext.erd [2.0.40]
2016-08-15 15:53:40.629 - > Start org.eclipse.draw2d [3.10.1.201508170204]

```

**Comment by serge-rider** [comment]
I need log with all errors, connection attempts and everything else in exact order it happens. 
Something fails somewhere and I can't figure out what is it..

**Comment by majkinetor** [comment]
I provideded complete content of `c:\Users\mmilic\.dbeaver\.metadata\dbeaver-debug.log `. Prior to it I deleted all logs.

The only other log is `.log`:

```
!SESSION 2016-08-15 16:12:47.549 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_77
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.ui 4 4 2016-08-15 16:12:51.017
!MESSAGE Unable to create part
!SUBENTRY 1 org.eclipse.ui 4 0 2016-08-15 16:12:51.017
!MESSAGE Factory org.jkiss.dbeaver.ui.editors.DatabaseEditorInputFactory returned null from createElement for editor id=org.jkiss.dbeaver.ui.editors.entity.EntityEditor name=null

```

**Comment by serge-rider** [comment]
I have no clue how to reproduce that..

For some reason DBeaver can't find schema object although it is definitely there.
The only thing I can suggest is to play with connection config/filters.

**Comment by majkinetor** [comment]
OK, its not a big issue, I can disable saving and error goes away. Its just strange that it happened on each computer (10+) at my company while its hard to reproduce. 

I could debug it myself when time allows...

**Comment by serge-rider** [comment]
Could you recheck this issue in 3.7.3? There were a couple of fixes in navigator model.

**Comment by majkinetor** [comment]
Still not working.

**Comment by Otiel** [comment]
I have the same issue, with version 3.7.3 and 3.7.4. Happens with all my connections, of type MS SQL (using Microsoft driver), PostgreSQL and SQLite. Exactly the same behavior as majkinetor:

* Schema viewers open with the following error:
`Factory org.jkiss.dbeaver.ui.editors.DatabaseEditorInputFactory returned null from createElement for editor id=org.jkiss.dbeaver.ui.editors.entity.EntityEditor name=null`

* SQL editors trigger the connection when the tab selected

**Comment by serge-rider** [comment]
I finally reproduced that.
DBeaver incorrectly searched for objects in non-active project.
Will be fixed in the next version.

**Comment by Otiel** [comment]
I confirm this is fixed for me in 3.7.5. Thanks for the quick resolution!
